packages/create-plugin/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,51 @@
+# Windows Troubleshooting
+
+Here are some things to try and check if you experience problems while working with create-plugin or sign-plugin in Windows.
+
+## I am getting: `Unsupported operating system 'Windows' detected. Please use WSL with create-plugin`
+
+create-plugin does not support native Windows. If you are using Windows, you can install WSL2 to work with create-plugin.
+
+## I am using Windows with WSL and I still get `Unsupported operating system 'Windows' detected.`
+
+Make sure you are using a WSL terminal. Simply installing WSL on Windows is not sufficient; you need to set up your WSL environment.
+
+To check if you are using a WSL terminal, run the following command:
+
+`uname -a`
+
+This command should return something like:
+
+`Linux ....-microsoft-standard-WSL2 #... GNU/Linux`
+
+If you receive an error or a different output, you are not using a WSL terminal.
+
+## I can confirm I am inside a WSL terminal, but I still get `Unsupported operating system 'Windows' detected.`
+
+You must install Node.js within your WSL environment. You can find a guide on how to install it [here](https://learn.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-wsl). We recommend installing the latest LTS version of Node.js.
+
+## I have Node.js installed inside WSL, but I still get `Unsupported operating system 'Windows' detected.`
```

Comment:
do you think we need this as a distinct point or can we merge the guidance on "what you can do" into the above point saying to install the latest LTS version of Node.js?